srcpkgs/v2ray/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'v2ray'
+pkgname=v2ray
+version=4.21.2
+revision=1
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Platform for building proxies to bypass network restrictions"
+maintainer="linarcx <linarcx@gmail.com>"
+license="MIT"
+homepage="https://github.com/v2ray/v2ray-core/"
+distfiles="${homepage}/releases/download/v${version}/${pkgname}-linux-64.zip"
+checksum=b0ce58976902092932afd26f894cabf0d47a48e32fed484a25d4f809e5611e8b
+
+do_install() {
+	unzip v2ray-linux-64.zip -d v2ray-linux-64
+	cd v2ray-linux-64
+
+	vbin v2ray
+	vbin v2ctl
+	vbin geoip.dat
+	vbin geosite.dat
+	vinstall ${FILESDIR}/config.json 644 etc/v2ray
```

Comment:
This is exactly upstream's `release/config/config.json`, so use upstream's please.